src/components/formItem/formItem.svelte

```diff
@@ -255,6 +255,8 @@
     --border-color: transparent;
     --border-color-hover: transparent;
     --shadow-hover: none;
+    --font: inherit;
+    --leo-icon-size: 1.43em;
```

Comment:
maybe a comment explaining where this magic number comes from (`20px default icon size / 14px default font-size`)

Reply:
```suggestion
    /**
     * This number is a ratio to preserve the proportions between
     * icon and text found the `normal` size:
     * 20px 'normal; icon size / 14px 'normal' font-size
     */
    --leo-icon-size: 1.43em;
```